packages/playwright-core/src/server/injected/domUtils.ts

```diff
@@ -14,6 +14,11 @@
  * limitations under the License.
  */
 
+let browserName = '';
```

Comment:
To emphasize we should not use it often:

```suggestion
let browserNameForWorkarounds = '';
```